src/reuse/modules/ui5/date.ts

```diff
@@ -3,6 +3,14 @@
 import { VerboseLoggerFactory } from "../../helper/verboseLogger";
 import { DateFormats } from "../util/constants/formatter.constants";
 
+type PickerMetadata = "sap.m.DatePicker" | "sap.m.DateTimePicker" | "sap.m.DateRangeSelection";
+
+class DatePickerSelectorParams {
+  selector: any;
```

Comment:
You can use the correct type: 
https://github.com/SAP/wdio-qmate-service/blob/main/src/reuse/modules/ui5/types/ui5.types.ts#L5

```suggestion
  selector: Ui5Selector;
```

Reply:
same everywhere

Reply:
done: https://github.com/SAP/wdio-qmate-service/pull/418/commits/a4de73df333b8f64dfb410cd3387923cbfb6a2b4